resolver/resolver.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 CloudWeGo Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resolver
```

Comment:
package github.com/kitex-contrib/registry-nacos/resolver

Reply:
If this package name must be like this. I think  we'd better the same with [registry-zookeeper/](https://github.com/kitex-contrib/registry-zookeeper/blob/main/registry/registry.go)

Reply:
oh, that my mistake. but if you use this, why you need to replace this package?